components/clinical/diagnosis-detail/src/index.tsx

```diff
@@ -34,90 +34,61 @@ const Seperator = styled.div`
   margin: 1rem 0;
 `
 
-const StyledDetail = styled.div`
+const StyledDetail = styled.div<IStyledViewport>`
   display: flex;
-  flex: 1;
-  justify-content: space-between;
-  padding-bottom: 1rem;
-`
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  margin-top: '1rem';
 
-const StyledColumn = styled.div`
-  display: flex;
-  flex: 1;
-  flex-direction: column;
-  padding-right: 0.5rem;
+  & dl {
+    width: ${({ isDesktop, isTablet }) => columns(isDesktop, isTablet)};
+    margin-top: 0;
+    margin-bottom: 1rem;
+
+    &:nth-child(4n + 1) {
+      text-align: ${({ isDesktop }) => (isDesktop ? 'start' : undefined)};
+    }
+
+    &:nth-child(4n + 2) {
+      text-align: ${({ isDesktop }) => (isDesktop ? 'center' : undefined)};
+    }
+
+    &:nth-child(4n + 3) {
+      text-align: ${({ isDesktop }) => (isDesktop ? 'center' : undefined)};
+    }
+
+    &:nth-child(4n) {
+      text-align: ${({ isDesktop }) => (isDesktop ? 'end' : undefined)};
+    }
+
+    &:nth-child(3n + 1) {
+      text-align: ${({ isTablet }) => (isTablet ? 'start' : undefined)};
+    }
+
+    &:nth-child(3n + 2) {
+      text-align: ${({ isTablet }) => (isTablet ? 'center' : undefined)};
+    }
+
+    &:nth-child(3n) {
+      text-align: ${({ isTablet }) => (isTablet ? 'end' : undefined)};
+    }
```

Comment:
This can all be done with css media queries (can be imported from @ltht-react/styles), something like the below:-

`
  ${DESKTOP_MEDIA_QUERY} {
      width: 25%;

      &:nth-child(4n + 1) {
        text-align: start;
      }

      &:nth-child(4n + 2),
      &:nth-child(4n + 3) {
        text-align: center;
      }

      &:nth-child(4n) {
        text-align: end;
      }
    }
`

Reply:
@Moza805 - Okay it might have been IE11 issues that was making it look off, I think the issue ended up being the text-align value of 'start' and 'end' instead of 'left' and 'right'.

Jonny sorted it out on this commit https://github.com/ltht-epr/ltht-react/pull/71/commits/55b6cc7731f5439d4ac605ebee6b05784e44339d